content/en/docs/concepts/workloads/pods/disruptions.md

```diff
@@ -227,6 +227,22 @@ can happen, according to:
 - the type of controller
 - the cluster's resource capacity
 
+## Pod disruption conditions {#pod-disruption-conditions}
+
+{{< feature-state for_k8s_version="v1.25" state="alpha" >}}
+
+When you enable the `PodDisruptionsCondition` [feature gate](/docs/reference/command-line-tools-reference/feature-gates/)
+in your cluster, a dedicated Pod `DisruptionTarget` condition is added to indicate
+a disruption. Its `reason` field additionally indicates one of the following
```

Comment:
```suggestion
an imminent disruption. Its `reason` field additionally indicates one of the following
```

Or some alternative way of saying that the disruption didn't happen yet, but it's about to happen.

Also, you might want to explain that the disruption might be interrupted (because of transient failures). The control plane might continue with the disruption, but its not guaranteed. After some time, the disruption target will be cleared if the conditions in the cluster changed.